ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -2,6 +2,92 @@
 // E2E Jenkins file.
 //
 
+import groovy.transform.Field
+
+@Field
+def podYAML = '''
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: main
+    image: gcr.io/k8s-testimages/kubekins-e2e:v20191108-9467d02-master
+    command:
+    - runner.sh
+    - sleep
+    - 99d
+    # we need privileged mode in order to do docker in docker
+    securityContext:
+      privileged: true
+    env:
+    - name: DOCKER_IN_DOCKER_ENABLED
+      value: "true"
+    resources:
+      requests:
+        memory: "8000Mi"
+        cpu: 8000m
+    # kind needs /lib/modules and cgroups from the host
+    volumeMounts:
+    - mountPath: /lib/modules
+      name: modules
+      readOnly: true
+    - mountPath: /sys/fs/cgroup
+      name: cgroup
+    # dind expects /var/lib/docker to be volume
+    - name: docker-root
+      mountPath: /var/lib/docker
+    # legacy docker path for cr.io/k8s-testimages/kubekins-e2e
+    - name: docker-graph
+      mountPath: /docker-graph
+  volumes:
+  - name: modules
+    hostPath:
+      path: /lib/modules
+      type: Directory
+  - name: cgroup
+    hostPath:
+      path: /sys/fs/cgroup
+      type: Directory
+  - name: docker-root
+    emptyDir: {}
+  - name: docker-graph
+    emptyDir: {}
+'''
+
+def build(SHELL_CODE) {
+	podTemplate(yaml: podYAML) {
+		node(POD_LABEL) {
+			container('main') {
+				def WORKSPACE = pwd()
+				dir("${WORKSPACE}/go/src/github.com/pingcap/tidb-operator") {
+					unstash 'tidb-operator'
+					stage("Debug Info") {
+						println "debug host: 172.16.5.5"
```

Comment:
172.16.5.5？

Reply:
yes, this is master node IP of Kubernetes cluster we're running on. I hardcoded it here for debugging purposes.